static/js/controllers/edit-user.js

```diff
@@ -37,9 +39,27 @@ var passwordTester = require('simple-password-tester'),
         $scope.enabledLocales = languages;
       });
 
-      var getType = function(roles) {
-        if (roles && roles.length) {
-          return roles[0];
+      var getRole = function(roles) {
+        if (!roles || !roles.length) {
+          return;
+        }
+        if (roles.indexOf(ADMIN_ROLE) !== -1) {
+          return ADMIN_ROLE;
+        }
+        if (!$scope.roles) {
+          // no configured roles
+          return;
+        }
+        // find all the users roles that are specified in the configuration
+        var knownRoles = roles.filter(function(role) {
+          return !!$scope.roles[role];
+        });
+        if (knownRoles.length) {
+          // Pre 2.16.0 versions stored the role we care about at the end
+          // of the roles array so for backwards compatibility return the
+          // last element.
+          // From 2.16.0 onwards users only have a one role.
```

Comment:
"a one role"

Also, interesting! It wasn't clear to me that users can only have one role. Are we doing that for now because it's simpler than allowing people to have multiple roles, or is there another reason.